app/views/plates/_plate_schematic.html.erb

```diff
@@ -0,0 +1,51 @@
+<%# locals: plate, well_values %>
+<%# plate: Plate %>
+<%# well_values: Array of [method, value] to call on well %>
+
+<%# Note: very similar to app/views/shared/_show_plate.html.erb %>
+
+<% descriptor = well_values[1].to_s %>
+
+<table class="plate mb-3" id="<%= descriptor %>">
+  <thead>
+    <tr>
+      <% num_rows = plate.height -%>
+      <% num_columns = plate.width -%>
+      <th></th>
```

Comment:
Does this header column need a css class?
Does it need to specify colspan attribute?